contracts/metatx/ERC2771ContextUpgradeable.sol

```diff
@@ -8,55 +8,75 @@ import "./IERC2771Upgradeable.sol";
 /**
  * @dev Implementing updatable Trusted forwarder
  */
-contract ERC2771ContextUpgradeable 
-    is Initializable, ContextUpgradeable, IERC2771Upgradeable {
+contract ERC2771ContextUpgradeable is
+  Initializable,
+  ContextUpgradeable,
+  IERC2771Upgradeable
+{
+  address private _trustedForwarder;
 
-    address private _trustedForwarder;
+  event TrustedForwarderChanged(address oldTF, address newTF);
 
-    event TrustedForwarderChanged(address oldTF, address newTF);
-
-    function __ERC2771_init(address forwarder) internal onlyInitializing {
-        if (forwarder != address(0)) {
-            _setTrustedForwarder(forwarder);
-        }
+  function __ERC2771_init(address forwarder) internal onlyInitializing {
+    if (forwarder != address(0)) {
```

Comment:
I understand that we named it "__ERC2771_init" to follow OZ naming but it is possible to remove this warning here ?

Reply:
done